src/parsers/component-parser.js

```diff
@@ -181,6 +178,27 @@ function parseComponent(
     htmlComponentContent
   ).matches.map(matchMapper(false));
 
+  const isGlobal = (style) => {
+    if (
+      style.component === 'Footer.abell' &&
+      style.attributes.global === true
+    ) {
+      return true;
+    }
+    return false;
+  };
+
+  const isFooter = styleMatches.every((styleMatch) => isGlobal(styleMatch));
+
+  if (
+    (options && !options.skipHTMLHash) ||
+    styleMatches.length > 0 ||
+    isFooter
+  ) {
+    // ignore adding scope hash
+    template = prefixHtmlTags(template, componentHash);
+  }
```

Comment:
```suggestion
  const shouldScopeTemplate = styleMatches.length <= 0 || styleMatches.every((styleMatch) => styleMatch.attributes.global === true);

  if (
    (options && !options.skipHTMLHash) ||
    styleMatches.length > 0 ||
    shouldScopeTemplate
  ) {
    // ignore adding scope hash
    template = prefixHtmlTags(template, componentHash);
  }
```

This code looks right to me. It checks if there are no style tags and checks if style tags exist then every style tag has global attribute